distributed/system_monitor.py

```diff
@@ -92,10 +93,6 @@ def update(self):
 
         # give external modules (like dask-cuda) a chance to initialize CUDA context
         if nvml is not None and nvml.nvmlInit is not None:
```

Comment:
I _think_ the `nvml.nvmlInit` check here is redundant now, though it shouldn't cause any problems to leave it here.

Reply:
It's not redundant, this is what I mentioned earlier when I also thought it was. It refers to the object in https://github.com/dask/distributed/blob/9d4f0bf2fc804f955a869febd3b51423c4382908/distributed/diagnostics/nvml.py#L5 and not to the `pynvml.nvmlInit` method. I think that's a confusing naming choice nevertheless but I won't touch it right now.

Reply:
I agree with that; I'm referring to the fact that by the time we call `update()`, we will have also called `nvml.one_time()`, meaning that `nvml.nvmlInit` will always not be `None` if `nvml` is not `None`.

Reply:
Ah sorry, you're right, good catch. I've updated that in https://github.com/dask/distributed/pull/4866/commits/79b315b660a1f045fbb7216d322a4b25a09bc801 .